web/src/cypress/support/a11y.ts

```diff
@@ -0,0 +1,21 @@
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      /** Test the accessibility of the current state of the page */
+      validateA11y: typeof validateA11y
+    }
+  }
+}
+
+// no selector provided will result in the entire page being checked
+// inject should be true once per page, after the cy.visit call
+function validateA11y(selector = 'main[id="content"]', inject = true): void {
+  if (inject) cy.injectAxe()
```

Comment:
Will this break if `validateA11y` is called multiple times on different components but the same page/test?

Reply:
it will if inject = true for all of them. that's a good callout, i think I'll set inject to `false` by default and the first time it's called it'll just need to set to true instead.

i originally had it this way since we call it just once in a bunch of tests more often than calling it multiple times on the same page for a given test

Reply:
Is there a global variable or something we can detect to make that automatic?